rails-commenting.rb

```diff
@@ -56,8 +61,10 @@ def blog_post_params
 
 # app/models/blog_post.rb
 
-# 9)
+# 9) Updating the class of this model to desribe it's association with another model
```

Comment:
Just about, eventually its updated, later on in the document, but for right now, it's just the definition of the class with inheritance.  The class can exist without associations.